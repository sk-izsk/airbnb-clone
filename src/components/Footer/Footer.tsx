import { Box, makeStyles, Typography } from '@material-ui/core';
import React from 'react';
import { CustomTheme } from '../../theme/muiTheme';

export interface FooterProps {}

const useStyles = makeStyles((theme: CustomTheme) => ({
  footer: {
    borderTop: `1px solid lightgray`,
    padding: theme.spacing(2.5),
    backgroundColor: '#f7f7f7',
    textAlign: 'center',
  },
  text: {
    fontSize: 14,
    padding: theme.spacing(0.625),
  },
}));

const Footer: React.FC<FooterProps> = () => {
  const classes = useStyles();
  return (
    <Box className={classes.footer}>
      <Typography className={classes.text} variant='body1'>
        © 2020 Airbnb clone! No rights reserved - this is a demo!
      </Typography>
      <Typography className={classes.text} variant='caption'>
        Privacy · Terms · Sitemap · Company Details
      </Typography>
    </Box>
  );
};

export { Footer };
