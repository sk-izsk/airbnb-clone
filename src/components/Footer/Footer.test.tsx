import { render } from '@testing-library/react';
import React from 'react';
import { Footer } from './Footer';

test('Footer renders correctly', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
