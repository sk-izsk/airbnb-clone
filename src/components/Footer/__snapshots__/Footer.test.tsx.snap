// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-3 makeStyles-footer-1"
  >
    <p
      class="MuiTypography-root makeStyles-text-2 MuiTypography-body1"
    >
      © 2020 Airbnb clone! No rights reserved - this is a demo!
    </p>
    <span
      class="MuiTypography-root makeStyles-text-2 MuiTypography-caption"
    >
      Privacy · Terms · Sitemap · Company Details
    </span>
  </div>
</DocumentFragment>
`;
